.github/workflows/go.yml

```diff
@@ -44,3 +44,19 @@ jobs:
 
     - name: test bindata/scripts
       run: make test-bindata-scripts
+
+  golangci:
+    name: Golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.17
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.8
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v3
+      with:
+        # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+        version: v1.37
```

Comment:
while we are at it, lets use a newer golangci version WDYT ?
latest is 1.46.1

Reply:
done